modules/nw-cudn-best-practices.adoc

```diff
@@ -28,4 +28,12 @@ Before setting up a `ClusterUserDefinedNetwork` custom resource (CR), users shou
 
 ** If the namespace is missing the `k8s.ovn.org/primary-user-defined-network` label and a primary `ClusterUserDefinedNetwork` CR already exists, a pod in the namespace is created and attached to the default network.
 
-** If the namespace _has_ the label, and a primary `ClusterUserDefinedNetwork` CR does not exist, a pod in the namespace is not created until the `ClusterUserDefinedNetwork` CR is created.
\ No newline at end of file
+** If the namespace _has_ the label, and a primary `ClusterUserDefinedNetwork` CR does not exist, a pod in the namespace is not created until the `ClusterUserDefinedNetwork` CR is created.
+
+* When using the `ClusterUserDefinedNetwork` CR to create `localnet` topology, the following are best practices for administrators:
+
+** You must make sure that the `spec.network.physicalNetworkName` field matches the one configured in the OVS bridge mapping when creating your CUDN. This ensures that you are bridging to the segment of your physical network that you intended to bridge to. Avoid reusing the same `physicalNetworkName` for different physical networks.
```

Comment:
I don't think "avoid" is the proper word to be used. 

In my understanding, the admin must be aware they are essentially *defining* how the network is when they use they attribute. I.e. if they re-use the `physicalNetworkName`, they are essentially **connecting** to that network. I.e. they are not "different physical networks" .